src/Branches.md

```diff
@@ -44,11 +44,13 @@ If you are interested in maintaining a new topic branch, then please [contact us
 
 ### `staging/rust-pci`
 
-[`staging/rust-pci`](https://github.com/Rust-for-Linux/linux/tree/staging/rust-pci) is dedicated to PCI-related abstractions, which are used by e.g. the [NVMe driver](NVMe-driver.md).
+[`staging/rust-pci`](https://git.sr.ht/~metaspace/linux-rust-pci) is dedicated to PCI-related abstractions, which are used by e.g. the [NVMe driver](NVMe-driver.md).
 
 The branch is kept in a compilable state (rebased regularly on top of `rust-next` or mainline). Fixes and features are welcome.
 
-It is maintained by Andreas Hindborg. Please contact him through [Zulip](Contact.md#zulip-chat).
+For general contact and for participating in development, please use the project's [mailing list](https://lists.sr.ht/~metaspace/linux-rust-pci)
```

Comment:
It is up to you how to develop and make people engage more and so on, but I imagine this means you will not use the issue tracker either and so on, which means it is not really what we discussed for "topic branches". It sounds more like a list of third-party projects. Which is fine, but we may need to move this to another list/section instead to differentiate it.

Reply:
The primary  contributors for `linux-rust-pci` prefer sourcehut over github. To accommodate their preferences I set up over at sourcehut, it is no problem for me. I kind of like the email list service they offer anyway.

However, I would prefer to keep the branch we have here on Github, so that people can file issues or PRs towards it and so that we keep the low barrier of entry that Github provides. I keep an eye out on any activity related to the branch here on Github.

I don't think it has to be an issue to have a presence on both platforms. Perhaps we can include both links on the website and tell people to work with whatever platform they want? Git is distributed anyway, so it does not really matter too much. For the email list versus the Github issues tracker, we have so little traffic that it does not matter for now.

@ojeda if you have any strong opinions against this the branch being mirrored, I think we should drop the Github one for now and keep stuff over at sourcehut, because that is what the contributors prefer.

Regarding the branch living in user owned repo (~metaspace) rather than something looking like an official org, I don't think that matters too much. People have kernel trees in all kinds of places, no reason we should not do so as well.

Reply:
To be clear, I don't particularly want to favor GitHub (or any other platform), and I definitely don't want to tell anybody what platform they should pick. If your current & future expected contributors want SourceHut, then it is a good call to use it.

However, there are two distinct issues I am trying to point out:

  - The first one (above) is the link going somewhere else. It is really just about the link: it is completely fine to have many mirrors, but the main link in the website in the list of topic branches should go to our mirror. For instance, I would definitely not expect if, in git.kernel.org, I click on one of the trees, and suddenly I am on GitHub for no reason (some of those trees keep mirrors too), i.e. it is not about the platform.

    As for the user-owned thing, it was also about the link, second-order: somehow the owner and the name of the tree/branch also changed when clicking on the link.

  - The second issue (this one) is that we will need to reassess how we spell this out in the website, because this is not what topic branches were meant to be, e.g. as you can see above in the file, the description would not make sense anymore. That is all fine, and we previously discussed having this sort of list, and in fact maybe we should start listing (or even mirroring) e.g. subsystem branches too (that are maintained), like DRM's and so on, rather than drop this one.

In summary, I don't care about platform X or Y, I just want to be clear about what we are doing under "topic branches".

Reply:
So, to be practical: with the main link kept as it was, I am happy to merge this already. I can fix the description of the topic branches later and we can rearrange/decide how to handle other topic branches later.

Reply:
Alright alright – I'd say linking both branches is a good idea. We tell people sth like "our primary development platform is sourcehut, but feel free to open issues on GitHub as well".
The only thing I want to insist on is that patches reach us through e-mail (this is a Kernel project, after all)

With that being said, my hope is that we can get upstream as soon as possible anyways. Then development would move to an official PCI kernel list.
So I perceive this here as a temporary thing anyways

Give me some time, I update the PR and provide a "v2"